8.0/s2i/bin/run

```diff
@@ -54,12 +54,15 @@ else
   echo "-> Cgroups memory limit is set, using HTTPD_MAX_REQUEST_WORKERS=${HTTPD_MAX_REQUEST_WORKERS}"
 fi
 
-if [ ! -v PHP_FPM_LOG_PATH ]; then
-  /bin/ln -s /dev/stderr ${PHP_FPM_LOG_PATH}/error.log
-fi
 
-chmod -R a+rwx ${PHP_FPM_LOG_PATH}
-chown -R 1001:0 ${PHP_FPM_LOG_PATH}
+
+if [ "x$PLATFORM" == "xel9" ] || [ "x$PLATFORM" == "xfedora" ]; then
+  if [ -z "${PHP_FPM_LOG_PATH:-}" ]; then
+    /bin/ln -s /dev/stderr ${PHP_FPM_LOG_PATH}/error.log
+  fi
+  chmod -R a+rwx ${PHP_FPM_LOG_PATH}
+  chown -R 1001:0 ${PHP_FPM_LOG_PATH}
```

Comment:
Shouldn't this command be in the above test about PHP_FPM_LOG_PATH being set

Reply:
You are right, it should be fixed commit https://github.com/sclorg/s2i-php-container/commit/122b25fe24807d4cc7aec24fdc62a262e78aa396